src/test/kotlin/cz/petrbalat/spring/mvc/test/dsl/controller/DslControllerTestV2.kt

```diff
@@ -102,28 +102,28 @@ class MockMvcDslBuilder(internal var url: String? = null,
 @RequestDsl
 class DslRequestBuilder(internal var url: String? = null,
```

Comment:
Removed. It was from an experiment with letting the URL be declared in the block, but since it is a parameter on the `request` call now. I'll remove it.